pages/booking/list.tsx

```diff
@@ -1,51 +1,54 @@
-import Alert from '@mui/material/Alert'
 import Booking from '../../components/booking'
-import hasNoUserAccess from '../../lib/utils'
-import prisma from '../../lib/prisma'
+import InfiniteScroll from "react-infinite-scroll-component"
+import Router from 'next/router'
+import { useEffect, useState } from 'react'
 
-function BookingList({ bookings }) {
-  if (!bookings) {
-    return <Alert severity="info">Loading...</Alert>
+function BookingList() {
+  const [bookingsLength, setBookingsLength] = useState(0)
+  const [bookings, setBookings] = useState([])
+  const [hasMore, setHasMore] = useState(true)
+  
+  useEffect(() => {
+    getMoreBookings() 
+  }, [])
+
+  async function getMoreBookings() {
+    fetch('/api/utils/get_more_data', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
```

Comment:
do we need content-type?

Reply:
This shouldn’t be a post request because we are not changing anything on the server.

Reply:
Also I think it’s fine if we’re only using url parameters instead of sending a body

Reply:
I used POST because I couldn't pass a parameter through body if it wasn't POST. so should I change to GET and pass the parameter via url?